packages/map-template/src/components/Search/Search.jsx

```diff
@@ -265,6 +283,23 @@ function Search({ onSetSize }) {
         }
     });
 
+    useEffect(() => {
```

Comment:
Add a comment explaining what we are doing here, and why.

Reply:
✅